packages/jsx-email/src/components/button.tsx

```diff
@@ -32,6 +32,7 @@ export const Button: JsxEmailComponent<ButtonProps> = ({
   fontSize = 16,
   align = 'left',
   withBackground = false,
+  rel,
```

Comment:
explicitly picked out `rel` since we do need to throw those onto the VML element too